### DISCUSSION TITLE
/recipe command not available

### DISCUSSION THREAD

**Discussion by Kolterdyx** [EssentialsX/Essentials#4122] [Q&A] [answered]
I am trying to use the `/recipe` command in my spigot server, that only has EX 2.18, but it says `This feature is not supported on the current server version`. I read through the issues here and found out that a `force-enable-recipe: true` feature exists, but when I tried adding it to the config.yaml, it did not work at all, and the error message still appears. Is there any way to enable the command?

**Comment by pop4959** [comment, marked as answer]
It's only supported on Paper on 1.15.2+; Spigot has a bug which allows users to duplicate items using the recipe book if /recipe is enabled.
